controllers/api/opennebula/machines.py

```diff
@@ -27,13 +27,16 @@ def index(self):
     @cherrypy.tools.isAuthorised(redirect=True)
     @cherrypy.tools.jinja(template="machines/history.html")
     def history(self):
-        pass
+        CLOUDPLATFORM = cherrypy.request.config.get("cloudPlatform")
+        return {"cloudPlatform" : CLOUDPLATFORM}
 
 
     @cherrypy.expose
     @cherrypy.tools.isAuthorised(redirect=True)
     @cherrypy.tools.jinja(template="machines/ssh.html")
     def ssh(self):
+        CLOUDPLATFORM = cherrypy.request.config.get("cloudPlatform")
```

Comment:
Doesn't this prevent `ssh()` returning a key?

Reply:
and the same thing happens under both OpenStack and OpenNebula?

Reply:
This is only used for jinja templating variables into the html of ssh page.

Reply:
This was fixed "offline" to this discussion thread, for both OpenStack and Open Nebula.